src/tests/interpolation/test_interpolation_structured2D.cc

```diff
@@ -292,17 +291,17 @@ CASE( "test_interpolation_structured using fs API for fieldset" ) {
 
     FieldSet fields_source;
     FieldSet fields_target;
-    using Value = float;
+    using Value = double;
```

Comment:
Can this test still be made to work with "float" ?
Or perhaps create another test with "double" ?